tests/data/use_braces_in_int_method_calls.py

```diff
@@ -0,0 +1,30 @@
+count = 5 .bit_count()
+abs = 10 .__abs__()
+is_integer = 10.5 .is_integer()
+
+if 10 .real: ...
+
+for number in range(1, 10):
+    if number.imag: ...
+
+exclusive_complex = 10j.imag
+
+hex_decimal = 0x10.real
```

Comment:
Should also test all the other kinds of numeric literals:
- octal
- binary
- scientific notation